README.md

```diff
@@ -1,2 +1,76 @@
-a lightweight hms runner in linux for wat-api
+
+A lightweight hms-runner for cloud wat (or cloud compute: `cc`)
+
+This library uses the [cc-java-sdk](https://github.com/USACE/cc-java-sdk) (see Note in Getting Started below) to read from a `payload` file stored in the bucket specified by `CC_AWS_S3_BUCKET` at the path `$CC_ROOT/$CC_MANIFEST_ID/payload`. In general, the `payload` provides a structured and consistent serialization of required data that CC can provide to plugins. In this case, the `payload` contains information for hms-runner to perform various `actions`.
+
+This library provides tools for running hms in a containerized environment with the following `Actions`:
+
+1. [downloadInputsAction](src/main/java/usace/cc/plugin/hmsrunner/downloadInputsAction.java): Verifies connection to S3 and downloads all files listed in the `payload` to the local directory (i.e. in the container) `/model/{model_name}` where `model_name` is specified in the payload as an attribute.
+2. [computeForecastAction](src/main/java/usace/cc/plugin/hmsrunner/computeForecast.java): Opens a project and runs a forecast for that project.
+3. [computeSimulationAction](src/main/java/usace/cc/plugin/hmsrunner/computeSimulationAction.java): Opens a project and runs a simulation for that project.
+4. [copyPrecipAction](src/main/java/usace/cc/plugin/hmsrunner/copyPrecipAction.java): Connects to a h5 instance to access hdf5 storage and copy files from a `source` to a `destination` as specified by the payload action.
+5. [dsstoCsvAction](src/main/java/usace/cc/plugin/hmsrunner/dsstoCsvAction.java): Reads a .dss file to write the `HecTimeSeries` to a destination csv file based on the datasource path
+6. [dsstoCHdfAction](src/main/java/usace/cc/plugin/hmsrunner/dsstoHdfAction.java): Reads a .dss file to write the `HecTimeSeries` to a destination csv file based on the datasource path
+7. [exportExcessPrecipAction](src/main/java/usace/cc/plugin/hmsrunner/exportExcessPrecipAction.java): Uses the hms model's `exportSpatialResults` function to export spatial variable data to a specified `destination`
+8. [pushOutputsAction](src/main/java/usace/cc/plugin/hmsrunner/pushOutputsAction.java): Takes the output files specified in the `payload` and pushes them to the S3 bucket.
+
+## Getting Started
+
+Note: To ensure the `cc-java-sdk` [submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules) is in place and up to date, run the `./update-gitsubmodule.sh` command before proceeding.
+
+### Requirements
+
+- [Docker](https://docs.docker.com/get-docker/)
+- AWS S3 (or minio running in local network)
+
+### Running the System
+
+Run the hms-runner with Docker:
+
+```
+docker build . -t <image_name>
+docker run --env-file .env-example <image_name>
+```
+
+Inside the dockerfile, the following line
+
+```
+ENTRYPOINT ["java", "-Djava.library.path=/HEC-HMS-4.11/bin/gdal:/HEC-HMS-4.11/bin", "-jar", "hms-runner-0.0.1.jar", "{\"root\":\"cc-store\", \"manifestID\":\"kanawha-test\"}"]
```

Comment:
Please add 3 examples of how to call this, starting with the original/default `.env`  approach